README.md

```diff
@@ -398,6 +398,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Divakar R](https://github.com/rexdivakar)
   - [Divash Gupta](https://github.com/Divxsh)
   - [Divlo](https://github.com/Divlo)
+  - [Divya](https://github.com/d-coder111)
```

Comment:
Please add your name in line 402, after Divya Raichura.


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix